source/blog/2018-11-09-the-ember-times-issue-72.md

```diff
@@ -41,8 +43,11 @@ And finally, read the full _RouteInfo Metadata_ proposal to learn more and leave
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [Premier League Video Tutorial ⚽️](https://twitter.com/gavinjoyce/status/1059533557130059779)
+
+[@GavinJoyce](https://github.com/gavinjoyce) made an awesome [tutorial](https://www.youtube.com/watch?time_continue=4&v=vYakop2d2RI) where he walks through how to build a live updating league table in the **latest Ember 3.5**! Code along with the video and you'll use [ember-cli-tailwind](https://github.com/embermap/ember-cli-tailwind) for styling, angle bracket components, `shuffle` from [ember-composable-helpers](https://github.com/DockYard/ember-composable-helpers) (TIL), [ember-math-helpers](https://github.com/shipshapecode/ember-math-helpers) and more.
```

Comment:
Looks like the YouTube link starts 4 seconds into the video, and I think "angle bracket components" should be "angle bracket invocation".
```suggestion
[@GavinJoyce](https://github.com/gavinjoyce) made an awesome [tutorial](https://www.youtube.com/watch?v=vYakop2d2RI) where he walks through how to build a live updating league table in the **latest Ember 3.5**! Code along with the video and you'll use [ember-cli-tailwind](https://github.com/embermap/ember-cli-tailwind) for styling, angle bracket invocation, `shuffle` from [ember-composable-helpers](https://github.com/DockYard/ember-composable-helpers) (TIL), [ember-math-helpers](https://github.com/shipshapecode/ember-math-helpers) and more.
```

Reply:
I updated those two small things just now in my European timezone afternoon-ness - I hope that's alright @amyrlam :sparkles:

Reply:
Yes, please feel free to commit to the branch anytime. Thanks for fixing that!